src/app/dialog-modal/dashboard-popup/dashboard-popup.component.ts

```diff
@@ -16,7 +16,8 @@ export class DashboardPopupComponent implements OnInit {
   }
   submit(){
     this.dialog.closeAll();
-    localStorage.setItem("role",JSON.stringify(this.selected))
+    //localStorage.setItem("role",JSON.stringify(this.selected))
+    localStorage.setItem("role",this.selected)
```

Comment:
Role is present in bearer token. Roles are of two types "EMPLOYER" or "EMPLOYEE". I am not sure do we need this right now. 

We can simply hardcode it now. When in future, we would be going for Employee Registration, we can have a look on this.